src/PortingAssistantExtensionClientShared/Utils/AwsUtils.cs

```diff
@@ -200,9 +203,10 @@ public static async Task<bool> ValidateProfileAsync()
                     ConfigurationFileName);
                 var TelemetryConfiguration = JsonConvert.DeserializeObject<PortingAssistantIDEConfiguration>(File.ReadAllText(ConfigurationPath)).TelemetryConfiguration;
 
-                if (awsCredentials == null || !await AwsUtils.VerifyUserAsync("", awsCredentials, TelemetryConfiguration))
+                if ((awsCredentials == null || !await AwsUtils.VerifyUserAsync("", awsCredentials, TelemetryConfiguration)) &&  isCredsNotificationDismissed) 
```

Comment:
Does it complain if we have AwsUtil.IsCredsNotificationDismissed? i think it will make it more clear that this is a global static.